build/yamls/externalnode/conf/antrea-agent.conf

```diff
@@ -0,0 +1,39 @@
+# FeatureGates is a map of feature names to bools that enable or disable experimental features.
```

Comment:
Probably change "externalnode" in the path to "vmnode".

Reply:
Again, the problem with "externalnode" is that it can refer to a node/policy receiver without Agent running.